packages/react-material-ui/src/components/FormLabel/index.tsx

```diff
@@ -21,7 +21,15 @@ const FormLabel = (props: Props) => {
   const { id, name, labelProps, label, required } = props;
 
   return (
-    <MuiFormLabel id={id} htmlFor={name}>
+    <MuiFormLabel
+      id={id}
+      htmlFor={name}
+      sx={{
+        '.MuiFormLabel-asterisk': {
```

Comment:
The asterisk only renders when `required === true`. This looks like a workaround and I believe it might be something wrong with the `required` prop.